lib/nifty_services/base_service.rb

```diff
@@ -188,7 +188,9 @@ def error!(status, message_key, options = {})
     end
 
     def valid_object?(record, expected_class)
-      record.present? && record.is_a?(expected_class)
+      blank = record.respond_to?(:empty?) ? !!record.empty? : !record
```

Comment:
Could be. But our looking for `spec/base_service_spec.rb:255` there are one test that check if the record isn't empty. I don't know the reason. The method `valid_object?` should check if the record is valid, and not if it's empty.

My suggestion is remove that test, because don't make sense.  And apply our suggestion over `record.is_a?(expected_class)`